specifications/EinsteinRiddle/Einstein.tla

```diff
@@ -146,4 +156,8 @@ Solution ==
 
 FindSolution == ~Solution
 
+\* To find the solution with the `^Apalache^' model-checker, run:
+\* apalache-mc check --init=Init --inv=FindSolution --length=0 Einstein.tla
+\* Then look for the file violation.tla in `^Apalache^' output directory.
```

Comment:
In case it is helpful, you can use the `--run-dir` flag to put the output (including the violation) in a known location:

```
 apalache-mc --help | grep 'run-dir'
   --run-dir            : additional directory wherein output files for this run will be written directly, default: none (overrides envvar RUN_DIR)
```